### PR TITLE
Fix/sidebar fixes

### PR DESCRIPTION
**Pull request by m0ksem**
👋

### CODE REVIEW

File: packages/docs/components/layout/Sidebar.vue [hunk 1]
```diff
@@ -168,8 +168,6 @@ export default defineComponent({
 @import "@/assets/smart-grid.scss";
```
Comment (asvae) [inline packages/docs/components/layout/Sidebar.vue:168]: Add breaking change to the log.

File: packages/ui/src/components/va-sidebar/VaSidebar.vue [hunk 2]
```diff
@@ -7,7 +7,7 @@
     @mouseenter="updateHoverState(true)"
     @mouseleave="updateHoverState(false)"
   >
-    <div class="va-sidebar__menu" :style="`width: ${computedWidth};`">
```
Comment (asvae) [inline packages/ui/src/components/va-sidebar/VaSidebar.vue:10]: Sidebar "right" story is broken (was before your pr as well).

Comment (asvae) [inline packages/ui/src/components/va-sidebar/VaSidebar.vue:10]: Sidebar "100% width" story is broken (aslwo was before your pr).

image

Comment (asvae) [inline packages/ui/src/components/va-sidebar/VaSidebar.vue:10]: Minimized width story is broken (is less than 25%)

image

Comment (asvae) [inline packages/ui/src/components/va-sidebar/VaSidebar.vue:10]: Hoverable minimized story seems to be broken (Should do somethin on hover? right now - does nothing)

image

**Review by asvae — CHANGES_REQUESTED** [review]
Too many stories are broken right now. I see docs are all right, but still.

Not saying we need them perfect (that's something to pursue in stories rework).

**Comment by m0ksem** [comment]
Because this PR makes breaking changes and all of this generally made for VaLayout component I'll merge this in feat/layout pr and close tihs PR.

**Pull request closed without merging**